.mocharc.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   extension: ['ts'],
   watchExtensions: ['ts'],
   spec: ['test/**/*.test.ts'],
-  timeout: 5000,
```

Comment:
can we override this setting only where it's needed? i guess in `target-starknet-test` package only